desktop/cmp/button/Button.scss

```diff
@@ -5,16 +5,376 @@
  * Copyright © 2020 Extremely Heavy Industries Inc.
  */
 
-.xh-button {
-  font-family: var(--xh-button-font-family);
-
-  &--autofocus-enabled {
-    // Hoist calls FocusStyleManager.onlyShowFocusOnTabs(), which by default hides focus indicators
-    // unless a component has been tabbed into via keyboard. Override here to always show a focus
-    // indicator for an autoFocus button so it's clear that it has focus.
-    &:focus {
-      outline: var(--xh-focus-outline) !important;
-      outline-offset: var(--xh-focus-outline-offset-px) !important;
+//------------------------
+// Core button styles (color-related styles in separate block below)
+//------------------------
+.xh-app,
+.xh-app.xh-dark,
+.xh-app.xh-dark.bp3-dark {
+  .xh-button.bp3-button {
+    border-radius: var(--xh-border-radius-px);
+    font-family: var(--xh-button-font-family);
+    font-size: var(--xh-button-font-size-px);
+    white-space: nowrap;
+
+    // Shaded/filled-in (non-minimal) mode.
+    &.xh-button--standard {
+      padding: var(--xh-pad-half-px) var(--xh-pad-px);
```

Comment:
Curious why standard has a wider horizontal padding than outlined? I can understand the rationale for reduced padding in minimal mode, but I would expect outlined and standard to take the same width.